### ISSUE TITLE
Downloader not working

### ISSUE THREAD

**Issue by dalgibbard** [tradel/appd-download-bot#3]
Hey Todd,
Having some trouble getting this to work:

```
[root@localhost orig]# download-appdynamics -u dgibbard@cisco.com -p XXXXXXXX -b64 java-agent
Checking available product versions . . ./usr/lib/python2.6/site-packages/bs4/__init__.py:166: UserWarning: No parser was explicitly specified, so I'm using the best available HTML parser for this system ("html.parser"). This usually isn't a problem, but if you run this code on another system, or in a different virtual environment, it may use a different parser and behave differently.

To get rid of this warning, change this:

 BeautifulSoup([your markup])

to this:

 BeautifulSoup([your markup], "html.parser")

  markup_type=markup_type))

Traceback (most recent call last):
  File "/usr/bin/download-appdynamics", line 10, in <module>
    main()
  File "/usr/lib/python2.6/site-packages/appd/download/download.py", line 159, in main
    username=args.username, password=args.password, debug=args.debug)
  File "/usr/lib/python2.6/site-packages/appd/download/zone.py", line 68, in __init__
    self.version = version
  File "/usr/lib/python2.6/site-packages/appd/download/zone.py", line 131, in version
    avail_versions = self.get_versions()
  File "/usr/lib/python2.6/site-packages/appd/download/zone.py", line 167, in get_versions
    dropdown = soup.find('table', id='version_disp').find('select')
AttributeError: 'NoneType' object has no attribute 'find'
Saving cookies to /root/.cookiejar . . .
```

Edited the code to fix the beautifulsoup error, but it still seems to be having issue parsing the `dropdown` element.

Any ideas/thoughts?

**Comment by dalgibbard** [comment]
FYI; this error appears to occur even if I put in invalid credentials - so looks like it's failing to authenticate, and then not getting the page it's expecting?

**Comment by tradel** [comment]
Fixed with commit edddd229d329dc9bc1d0d46e6aa07bdea3ac5b8f.
